packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -68,8 +68,9 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     const schemaComposer = new SchemaComposer(schema)
 
     this.buildSchemaModelRelationships(schemaComposer, models);
-
+    
     this.buildSchemaForModels(schemaComposer, models);
+    this.addMetaFields(schemaComposer, models);
```

Comment:
Ah but this would mean these fields will automatically get added to the mutation input types, which we don't want.

I think it is fine link this for now, but in future schemaDefinitions should detect and filter out a predefined set of fields that should not be included in mutation inputs

https://github.com/aerogear/graphback/blob/c6f7fba82b107fd2c09075362e8fc7ee9582898d/packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts#L99